app/src/main/res/values/colors.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="colorPrimary">#008577</color>
-    <color name="colorPrimaryDark">#00574B</color>
-    <color name="colorAccent">#D81B60</color>
+    <color name="color_primary">#393939</color>
+    <color name="color_primary_dark">#2f2f2f</color>
+    <color name="color_accent">#52b7f8</color>
+    <color name="background">#2b2b2b</color>
```

Comment:
Em nên dùng với tên của color: http://chir.ag/projects/name-that-color/#2B2B2B
```suggestion
    <color name="color_mine_haft">#2b2b2b</color>
```
tương tự vơi 2 dòng 7. 8